recipes/gmsh/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "gmsh" %}
+{% set version = "3.0.6" %}
+{% set sha256 = "9700bcc440d7a6b16a49cbfcdcdc31db33efe60e1f5113774316b6fa4186987b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}-source.tar.gz
+  url: http://gmsh.info/src/{{ name }}-{{ version }}-source.tgz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: mkdir build && cd build && cmake .. && make && make install
```

Comment:
you probably want to have `cmake -DCMAKE_INSTALL_PREFIX=$PREFIX ..`

Reply:
Which is the correct install location that is writable?

Reply:
It's just $PREFIX :)